Client/mods/deathmatch/logic/rpc/CColShapeRPCs.cpp

```diff
@@ -19,6 +19,7 @@ void CColShapeRPCs::LoadFunctions(void)
     AddHandler(UPDATE_COLPOLYGON_POINT, UpdateColPolygonPoint, "UpdateColPolygonPoint");
     AddHandler(ADD_COLPOLYGON_POINT, AddColPolygonPoint, "AddColPolygonPoint");
     AddHandler(REMOVE_COLPOLYGON_POINT, RemoveColPolygonPoint, "RemoveColPolygonPoint");
+    AddHandler(COLSHAPE_POLYGON_SET_HEIGHT, SetColShapePolygonHeight, "SetColShapePolygonHeight");
```

Comment:
Perhaps rename to `SET_COLPOLYGON_HEIGHT` for consistency.
```suggestion
    AddHandler(SET_COLPOLYGON_HEIGHT, SetColShapePolygonHeight, "SetColShapePolygonHeight");
```